docs/flow/modin/pandas/series.rst

```diff
@@ -1,7 +1,10 @@
 :orphan:
 
+======================
 Series Module Overview
-""""""""""""""""""""""
+======================
+
+.. currentmodule:: modin.pandas
```

Comment:
```suggestion
.. py:currentmodule:: modin.pandas
```
Isn't py: prefix required here? Also, why should we have this?

Reply:
It works as is, so i think `py:` directive can be omitted. Moreover this change will be moved to separate PR and it is in irrelevant now.